capstone.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Electrical Engineering Portal</title>
+    <link rel="stylesheet" href="capstone.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css">
+</head>
+
+
+
+<header>
+    <nav class="container">
+        <div class="upper-nav">
+                <a href="https://www.facebook.com/haftamudesta"><i class="fa-brands fa-4x fa-facebook"></i></a> 
+            <i class="fa-brands fa-twitter-square"></i>
+            <p>English</p>
+            <p>My page</p>
+            <p>Logout</p>
+        </div>
+        <div class="lower-nav">
+            <div class="logo">
+                <a href="capstone.html"><img src="images/electric-power-engineers1.png" alt=""></a>
+                <img src="images/Enabled.png" class="hamburger" alt="hambergur menu">
+                <p class="close">&times;</p>
+            </div>
+            <div class="information">
+                <ul>
+                    <li>
+                        <a href="about.html" class="is-active" id="information-active">About</a>
+                    </li>
+                    <li>
+                        <a href="#" class="is-actives">Program</a>
+                    </li>
+                    <li>
+                        <a href="#" class="is-actives">Join</a>
+                    </li>
+                    <li>
+                        <a href="#" class="is-actives">Sponser</a>
+                    </li>
+                    <li>
+                        <a href="#" class="is-actives">News</a>
+                    </li>
+                </ul>
+            </div>
+            <div class="company">
+                <button>EEP Taining</button>
+            </div>
+
+        </div>
+    </nav>
+</header>
+    <div class="main">
+        <div class="hello" id="home">
+            <p class="heading">"Hello! Engineers in the world"</p>
+            <h2>Welcome to Electrical <br> enginering portal  launches <br>on Fri,july 2022</h2>
+            <p class="introduction">Power engineering, also called power systems engineering, is a subfield of electrical engineering that deals with the generation, transmission, distribution, and utilization of electric power, and the electrical apparatus connected to such systems.this seminar is a gate way  to power engineering.</p>
+            <h4>2022.7.29(FRI)~5(THU)</h4>
+            <p class="hall"><span>&#64;</span>Addis Ababa institute of technology,center of research and technology</p>
+        </div>
+        <div class="main-catagories" id="about">
+            <h2 class="under-line">main catagories</h2>
```

Comment:
- [ ] Please align your content properly as requested by the previous reviewer. See below screenshots 👇 
- Your content 👇 
![2](https://user-images.githubusercontent.com/67087664/181904664-11d1e61a-c6fe-4d33-8fc5-f894589cf0e7.png)
- Original 👇 
![-2](https://user-images.githubusercontent.com/67087664/181904675-cfc988c1-d9e1-46d7-b35d-44f46e72318a.png)